datatree/tests/test_datatree.py

```diff
@@ -454,8 +447,41 @@ def test_roundtrip_unnamed_root(self):
         assert roundtrip.equals(dt)
 
 
-class TestBrowsing:
-    ...
+class TestDatasetView:
+    def test_view_contents(self):
+        ds = create_test_data()
+        dt = DataTree(data=ds)
+        assert ds.identical(
+            dt.ds
+        )  # this only works because Dataset.identical doesn't check types
```

Comment:
These asserts do pass because for some reason `ds.identical(other)` doesn't explicitly check types are the same, even though `xarray.testing.assert_identical` does check types :confused: